### PR TITLE
Add LICENSE file

### PR DESCRIPTION
**Pull request by brikis98**
This repo didn't have one!

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Copyright (c) 2016 Gruntwork, Inc. All rights reserved.
+info@gruntwork.io
```
Comment (ina-stoyanova) [inline LICENSE.txt:2]: Did you mean to add the same info as in the other repos?

Comment (ina-stoyanova) [reply LICENSE.txt:2]: e.g. https://github.com/gruntwork-io/fetch-test-public/pull/2/files

**Review by ina-stoyanova — APPROVED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (brikis98) [reply LICENSE.txt:2]: The other repos are using open source licenses. This repo is not meant to use an open source license.

Comment (ina-stoyanova) [reply LICENSE.txt:2]: Aha makes total sense! Thank you :)

**Comment by brikis98** [comment]
Thanks for the review! Merging now.

**Pull request merged**